Neos.Neos/Classes/Service/Controller/AbstractServiceController.php

```diff
@@ -110,6 +110,7 @@ public function processRequest(ActionRequest $request, ActionResponse $response)
      */
     protected function convertException(\Exception $exception)
     {
+        $exceptionData = [];
```

Comment:
line 146 _could_ in a very unlikely case throw an Error because the variable is not initialised.

Also line 116 makes use of phps implicit variable array initialisation which is unexpected ^^